src/components/settings-menu/BackupSection/ShowSecretView.js

```diff
@@ -0,0 +1,165 @@
+import { useRoute } from '@react-navigation/native';
+import React, { useEffect, useState } from 'react';
+import ShadowStack from 'react-native-shadow-stack/dist/ShadowStack';
+import styled from 'styled-components';
+import WalletTypes from '../../../helpers/walletTypes';
+import { useClipboard, useDimensions, useWallets } from '../../../hooks';
+import {
+  identifyWalletType,
+  loadSeedPhraseAndMigrateIfNeeded,
+} from '../../../model/wallet';
+import { colors, position } from '../../../styles';
+import { ButtonPressAnimation } from '../../animations';
+import { FloatingEmojis } from '../../floating-emojis';
+import { Icon } from '../../icons';
+import { Centered, Column, Row, RowWithMargins } from '../../layout';
+import { Text } from '../../text';
+
+const PrivateKeyText = styled(Text).attrs({
+  align: 'center',
+  color: 'dark',
+  letterSpacing: 0.6,
+  lineHeight: 'looser',
+  size: 'lmedium',
+  weight: 'semibold',
+})`
+  padding-left: 30;
+  padding-right: 30;
+`;
+
+const Shadow = styled(ShadowStack)`
+  elevation: 15;
+  margin-bottom: 85;
+  margin-top: 19;
+`;
+
+const ShowSecretView = () => {
+  const { setClipboard } = useClipboard();
+  const { params } = useRoute();
+  const { selectedWallet } = useWallets();
+  const [seed, setSeed] = useState(null);
+  const [type, setType] = useState(null);
+  const { width: deviceWidth } = useDimensions();
+  let wordSectionHeight = 100;
+
+  useEffect(() => {
+    const loadSeed = async () => {
+      const wallet_id = params?.wallet_id || selectedWallet.id;
+      const s = await loadSeedPhraseAndMigrateIfNeeded(wallet_id);
+      const walletType = identifyWalletType(s);
+      setType(walletType);
+      setSeed(s);
+    };
+    loadSeed();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  let columns = [];
+  let secretLayout = null;
+  if (seed && type === WalletTypes.mnemonic) {
+    wordSectionHeight = (seed && (seed.split(' ').length || 12) / 2) * 39 + 10;
+    const words = seed.split(' ');
+    columns = [words.slice(0, words.length / 2), words.slice(words.length / 2)];
+    secretLayout = columns.map((wordColumn, colIndex) => (
+      <Column
+        // eslint-disable-next-line react/no-array-index-key
```

Comment:
can we maybe remove this rule? I don't see value of it

Reply:
I agree! just did it